client/post-editor/editor-ground-control/index.jsx

```diff
@@ -370,18 +291,14 @@ const EditorGroundControl = React.createClass( {
 						{ this.getPreviewLabel() }
 					</button>
 					<div className="editor-ground-control__publish-combo">
-						<button
-							className="editor-ground-control__publish-button button is-primary"
-							onClick={ this.onPrimaryButtonClick }
-							disabled={ ! this.isPrimaryButtonEnabled() }
+						<EditorPublishButton
+							{ ...this.props }
 							tabIndex={ 5 }
-						>
-							{ this.getPrimaryButtonLabel() }
-						</button>
-						{ siteUtils.userCan( 'publish_posts', this.props.site ) &&
+						/>
+						{ this.canSchedulePost() &&
 							<button
 								ref="schedulePost"
-								className="editor-ground-control__time-button button is-primary"
```

Comment:
Hmmm... I don't see the value of that over applying the `@extend .is-primary` in CSS.

Note that the horrible `@mixin` hack is *not* on the `EditorPublishButton`, the `EditorPublishButton` is primary or not depending on what its container is (mobile navigation bar or GroundControl), and that can be implemented in CSS.